src/main/resources/application.properties

```diff
@@ -1,22 +1,23 @@
 #Server properties
 server.port=9004
 #Datasource properties
-spring.datasource.driverClassName=com.mysql.jdbc.Driver
-spring.datasource.url=jdbc:mysql://localhost:3306/login_db
+spring.datasource.driverClassName=com.mysql.cj.jdbc.Driver
+spring.datasource.url=jdbc:mysql://localhost:3306/login_db?createDatabaseIfNotExist=true&useUnicode=true&useJDBCCompliantTimezoneShift=true&useLegacyDatetimeCode=false&serverTimezone=Asia/Jakarta&useSSL=false&allowPublicKeyRetrieval=true
 spring.datasource.username=root
 spring.datasource.password=pass
 spring.datasource.testWhileIdle=true
 spring.datasource.validationQuery=SELECT 1
 #JPA properties. Using ddl-auto = create will drop schema every-time.
 #Choose the correct property based on development / production role.
-spring.jpa.hibernate.ddl-auto=create
-spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.MySQL5InnoDBDialect
-spring.datasource.initialization-mode=always
+spring.jpa.hibernate.ddl-auto=update
+spring.jpa.generate-ddl=true
+spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.MySQL8Dialect
+#spring.sql.init.mode=always
 spring.jpa.properties.hibernate.format_sql=true
 logging.level.org.hibernate.SQL=DEBUG
 #JWT properties
 app.jwt.header=Authorization
-app.jwt.header.prefix=Bearer 
+app.jwt.header.prefix=Bearer
```

Comment:
There is a space after "Bearer " that you have trimmed that will break authentication. The JWT token always starts with "Bearer" + " " 

Do not touch unrelated properties in this diff/PR.

Reply:
ok